holidays/countries/new_zealand.py

```diff
@@ -78,9 +76,14 @@ class NewZealand(HolidayBase):
         "WTL",  # Correct code is WTC
     ]
 
-    def _populate(self, year):
-        super()._populate(year)
+    @staticmethod
```

Comment:
It seems like a good candidate for DRY optimization (saw similar code in canada.py). No action required, just saying.

Reply:
Sorry, I didn't quite understand what it was about :(

Reply:
We're going to have exactly the same code in 2 different places. Could be extracted and reused.

Reply:
You mean `_get_nearest_monday`, I got it! :) Should I move it to utils.py?